src/components/Graph/MainGraph.js

```diff
@@ -71,15 +70,16 @@ class MainGraph extends Component {
     initialize = (dataset) => {
         // initialize() trigged on mount and Dataset change
         const { indicators } = this.props;
-        const { dateThreshold } = this.state;
-
 
         // SCENARIOS: various scenario variables used for a given geoid
         const SCENARIOS = buildScenarios(dataset);  
         const scenarioMap = buildScenarioMap(dataset);
         const firstScenario = SCENARIOS[0];
         const firstIndicator = indicators[0];
         const firstSeverity = scenarioMap[firstScenario.key][0];
+        // '2020-07-19-21-44-47-inference'
+        const dateString = firstScenario.key.substring(0,10)
```

Comment:
We should remember to add in the validator script to check the first 10 substring is equal to a date